sdk/python/kfp/client/client.py

```diff
@@ -1132,6 +1137,17 @@ def create_run_from_pipeline_package(
             enable_caching=enable_caching,
             service_account=service_account,
         )
+
+        link = f"{self._get_url_prefix()}/#/experiments/details/{experiment.id}"
+        if self._is_ipython():
+            import IPython
+            html = f'<a href="{link}" target="_blank" >Run details</a>.'
+            IPython.display.display(IPython.display.HTML(html))
+        else:
+            print(f"Run details: {link}")
```

Comment:
experiment details?

Reply:
Also, probably want to show run URL here, not experiment, right?

Reply:
Yikes! Good catch, @connor-mccarthy. Thank you! Fixed.